website/docs/r/ecrpublic_repository.html.markdown

```diff
@@ -10,10 +10,19 @@ description: |-
 
 Provides a Public Elastic Container Registry Repository.
 
+~> **NOTE:** This resource can only be used with `us-east-1` region.
+
 ## Example Usage
 
 ```terraform
+provider "aws" {
+  alias   = "us_east_1"
+  region  = "us-east-1"
+}
+
 resource "aws_ecrpublic_repository" "foo" {
+  provider = aws.us_east_1
+  
```

Comment:
Hi @Vrtak-CZ, these are the changes you need (whitespace issues) and will get the linter passing. I would have made them myself but I can't push commits to your org.

```suggestion
provider "aws" {
  alias  = "us_east_1"
  region = "us-east-1"
}

resource "aws_ecrpublic_repository" "foo" {
  provider = aws.us_east_1

```

Reply:
omg my vscode config messed it up again :-/ should be fixed now sorry